src/index.js

```diff
@@ -23,8 +31,8 @@ const mascot = new MascotBot({
     {
       behavior: DaysUntil,
       settings: {
-        conDate: '8/11/2017',
-        maxDays: 398,
+        conDate: '2018-07-26 09-05:00',
```

Comment:
This isn't a valid date. Maybe stick to a simpler format.

Reply:
Our local test disagrees '-'
That's _not quite_  ISO 8601, but I believe the library which supports parsing those to times was complaining that the _previous_ version was going to be deprecated, so we saw to it to change it to this form. (It's missing MM:SS and T, isn't it :)
ISO 8601 is pretty unambiguous, would you prefer it in proper form instead?

Reply:
Sorry, I didn't realize that was ISO-8601, as long as moment has no problems with it!

Reply:
Yeah, when I went to run the countdown behaviour for the first time. I encountered a Depreciation Warning in console. It worked, it just threw a warning out, so I tweaked the format a bit in order to keep a clean console